src/components/ForgotPassword/ForgotPassword.jsx

```diff
@@ -32,107 +90,141 @@ export default function ResetPassword() {
+               <StyledCaption>
+                  Please enter the email that your RISE volunteer account is
+                  associated with.
+               </StyledCaption>
+               <StyledSubtitle>Email</StyledSubtitle>
                <Form.Control
+                  className="email-box"
                   autoFocus
                   type="email"
                   value={username}
                   onChange={(e) => setUsername(e.target.value)}
                />
             </Form.Group>
-            <Button
-               type="submit"
-               isLoading={isSendingCode}
-               disabled={!username}
-            >
-               Send Confirmation
-            </Button>
-         </Form>
+            <SubmitButton type="submit" isLoading={isSendingCode}>
+               Submit
+            </SubmitButton>
+         </StyledForm>
       );
    }
 
    function renderConfirmationForm() {
       return (
-         <form onSubmit={handleConfirmClick}>
+         <StyledForm onSubmit={handleConfirmClick}>
+            <div className="title-container">
+               <StyledTitle>Forgot password</StyledTitle>
+            </div>
             <Form.Group bsSize="large" controlId="code">
-               <Form.Label>Confirmation Code</Form.Label>
+               <StyledCaption>
+                  An email has been sent to ({username}) to verify you are
+                  trying to change your password.
+               </StyledCaption>
+               <StyledSubtitle>
+                  Please enter the verification code:
+               </StyledSubtitle>
                <Form.Control
                   autoFocus
                   type="tel"
                   value={code}
                   onChange={(e) => setCode(e.target.value)}
                />
-               <div>
-                  Please check your email ({username}) for the confirmation
-                  code.
+            </Form.Group>
+            <SubmitButton type="submit" isLoading={isConfirming}>
+               Verify
+            </SubmitButton>
+         </StyledForm>
+      );
+   }
+
+   function renderNewPasswordForm() {
+      return (
+         <StyledForm onSubmit={handlePasswordSetClick}>
+            <Form.Group bsSize="large" controlId="code">
+               <div className="title-container">
+                  <StyledTitle>Change password</StyledTitle>
                </div>
+               <StyledCaption>
+                  Password reset has been verified. Please set a new password.
+               </StyledCaption>
             </Form.Group>
-            <hr />
             <Form.Group bsSize="large" controlId="password">
-               <Form.Label>New Password</Form.Label>
+               <StyledSubtitle>New Password</StyledSubtitle>
                <Form.Control
                   type="password"
                   value={password}
                   onChange={(e) => setPassword(e.target.value)}
                />
```

Comment:
Need to add show/hide password functionality. Check @ObjectJosh and @SamuelSehnert PR's for references.